src/main/java/org/jabref/gui/util/FileFilterConverter.java

```diff
@@ -62,11 +64,16 @@ private FileFilterConverter() {
     }
 
     public static FileFilter toFileFilter(FileChooser.ExtensionFilter extensionFilter) {
-        List<String> extensionsCleaned = extensionFilter.getExtensions()
-                                                        .stream()
-                                                        .map(extension -> extension.replace(".", "").replace("*", ""))
-                                                        .filter(StringUtil::isNotBlank)
-                                                        .collect(Collectors.toList());
+        return toFileFilter(extensionFilter.getExtensions());
+    }
+
+    // Implement as DirectoryStream https://docs.oracle.com/javase/9/docs/api/java/nio/file/DirectoryStream.Filter.html
+
+    public static FileFilter toFileFilter(List<String> extensions) {
+        List<String> extensionsCleaned = extensions.stream()
```

Comment:
Is it not possible to reuse the `toDirFilter` here? They are almost the same code. Something like
```java
filter = toDirFilter(extensions);
return file -> filter.apply(file.toPath())
```